h2/test_h2_specs.py

```diff
@@ -102,3 +102,40 @@ def test_h2_specs(self):
         self.start_tempesta()
         self.start_all_clients()
         self.wait_while_busy(h2spec)
+
+class H2Load(tester.TempestaTest):
+    '''Tests for h2 proto implementation. Run h2load utility against Tempesta.
+    Simply check return code and warnings in system log for test errors.
+    '''
+
+    clients = [
+        {
+            'id' : 'h2load',
+            'type' : 'external',
+            'binary' : 'h2load',
+            'ssl' : True,
+            'cmd_args' : ' https://${tempesta_ip} -D100'
```

Comment:
Duration is configured in tests configuration file. Better to follow that value and avoid hardcoded ones. Unless hardcoded test duration if really required, e.g. to stay in sync with some OS/application timers.

Reply:
There is a dependence on time. The duration from the config may be small